### PR TITLE
chore: market stats tests

### PR DESCRIPTION
**Pull request by pepopowitz**
The type of this PR is: **Tests**

This PR partially resolves [CX-738]

### Description

This PR adds tests for the MarketStats component. 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-738]: https://artsyproduct.atlassian.net/browse/CX-738

### CODE REVIEW

File: src/lib/Components/Artist/ArtistInsights/__tests__/MarketStats-tests.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      }
+
+      mockEnvironmentPayload(environment, {
+        PriceInsightConnection: () => priceInsights,
+      })
+
+      return tree
+    }
+
+    it("displays down arrow when percentage is under 100", () => {
```
Comment (pepopowitz) [inline src/lib/Components/Artist/ArtistInsights/__tests__/MarketStats-tests.tsx:80]: These tests [are validating incorrect behavior](https://artsyproduct.atlassian.net/browse/CX-1003). I'm going to correct them in a followup PR.

Comment (pepopowitz) [reply src/lib/Components/Artist/ArtistInsights/__tests__/MarketStats-tests.tsx]: I merged the followup PR into here, so these tests are now validating the correct behavior.

File: src/lib/Components/Artist/ArtistInsights/__tests__/MarketStats-tests.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    it("shows data for other mediums when selected", () => {
+      tree.findByProps({ title: "Select medium" }).props.onSelectValue("fingerpaint")
+
+      expect(extractText(tree.findByProps({ "data-test-id": "annualLotsSold" }))).toEqual("456")
+    })
+  })
+
+  describe("median sale over estimate icons", () => {
+    function renderWithOnePriceInsightNode(
+      nodeSpecs: Partial<NonNullable<NonNullable<MarketStats_priceInsightsConnection["edges"]>[0]>["node"]>
```
Comment (MounirDhahri) [inline src/lib/Components/Artist/ArtistInsights/__tests__/MarketStats-tests.tsx:60]: 👏

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
LGTM, feel free to merge once Adam takes a look at this as well!

**Review by admbtlr — APPROVED** [review]
💯

**Comment by admbtlr** [comment]
Needs some conflict resolution before merging

**Pull request merged**